src/registry_schemas/schemas/incorporation_application.json

```diff
@@ -51,8 +49,40 @@
                 },
                 "nameTranslations": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_translations"
+                },
+                "cooperative": {
```

Comment:
change this so it's its own section like nameRequest
and then keep it in alpha order